source/usage-examples/findOne.txt

```diff
@@ -0,0 +1,70 @@
+.. _kotlin-usage-findone:
+
+===============
+Find a Document
+===============
+
+.. default-domain:: mongodb
+
+You can retrieve a single document in a collection by chaining together
+the ``find()`` and ``first()`` methods on a ``MongoCollection`` object.
+You can pass a query filter to the ``find()`` method to query for and
+return documents that match the filter in the collection. If you do not
+include a filter, MongoDB returns all the documents in the collection.
+
+For more information on querying MongoDB with the Kotlin driver, see our
+:doc:`guide on Querying Documents </fundamentals/crud/read-operations/retrieve/>`.
+
+You can also chain other methods to the ``find()`` method
+such as ``sort()`` which organizes the matched documents in a specified order, and 
+``projection()`` which configures the fields included in the returned documents.
+
+For more information on the ``sort()`` method, see our
+:doc:`guide on Sorting </fundamentals/crud/read-operations/sort/>`.
+For more information on the ``projection()`` method, see our
+:doc:`guide on Projections </fundamentals/crud/read-operations/project/>`
+
+The ``find()`` method returns an instance of ``FindFlow``, a class
+that offers several methods to access, organize, and traverse the results.
+.. TODO (DOCSP-29176) Update with accurate inheritance information when available
+.. ``FindFlow`` also inherits methods from its parent class ``Flow`` from the
+.. Kotlin Coroutines library, such as ``first()`` and ``firstOrNull()``.
+
+The ``firstOrNull()`` method returns the first document from the retrieved results
+or ``null`` if there are no results. The ``first()`` method returns 
+the first document or throws a ``NoSuchElementException`` exception if no 
+documents match the query.
+
+.. TODO (DOCSP-29176) Add link to Flow documentation when available
+.. For more information on accessing data from a flow with the Kotlin driver, see our
+.. :doc:`guide on Accessing Data From a Flow </fundamentals/crud/read-operations/flow/>`.
+
+Example
+-------
+
+The following snippet finds a single document from the ``movies`` collection.
+It uses the following objects and methods:
+
+- A **query filter** that is passed to the ``find()`` method. The ``eq``
+  filter matches only movies with the title exactly matching the text
+  ``'The Room'``.
```

Comment:
```suggestion
  ``"The Room"``.
```

Reply:
fix elsewhere too if possible to match the way strings appear in the code examples